packages/ui/src/components/vuestic-components/va-select/VaSelect.vue

```diff
@@ -436,6 +463,18 @@ export default class VaSelect extends mixins(
 
   showDropdown () {
     this.doShowDropdownContent = true
+    this.scrollToSelected()
+  }
+
+  scrollToSelected () {
+    if (!this.valueComputed.length) {
+      return
+    }
+
+    const selected = this.valueComputed
+    const scrollTo = typeof selected === 'string' ? selected : selected[selected.length - 1]
+    this.hoveredOption = scrollTo
+    this.$nextTick(() => (this.$refs as any).optionList.scrollToOption(scrollTo))
```

Comment:
Minor thing, but you can declare `@Ref('optionList')`, then you don't have to use `any`.

Reply:
Thanks.

I've tried to use our decorator for refs but if I do something like this:

- Declare up top `@Ref('optionList') optionListRef: any`
- `... this.$nextTick( this.optionListRef.scrollToOption(scrollTo) )`

It works but there are some cases where ref points to null.

We could declare it within the next tick as well but it kind of overcomplicates logic and kills the whole point of using it.

Correct me if I'm wrong, maybe I missing something.

Reply:
we might not have a way to do without nexttick, my point was only about ref